multiqc_cmgg/modules/msi_sensor_pro/msi_sensor_pro.py

```
@@ -58,27 +69,196 @@ def __init__(self):
+        # summary table - also convert to HTML to avoid violin warnings
+        try:
+            # Sort summary data: primary by perc (descending), secondary by low_coverage_sites (ascending)
+            sorted_samples = sorted(
+                data_dicts_summary.items(),
+                key=lambda x: (-x[1].get('perc', 0), x[1].get('low_coverage_sites', 0))
+            )
+            
+            # Build table with sortable columns
+            table_id = "msi-summary-table"
+            html_parts = [
+                f'<style>',
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)